common/info.yaml

```
@@ -1292,6 +1292,128 @@ description: |
+  say pure red, the device has only a single physical LED channel to contribute 
+  to the overall physical brightness (value) of the light. The brightest
+  pure colors are along the top of the tent wall (the top circumferance of
+  the cylinder). However, when generating white light, the device can bring 
+  to bare all three (or four) channels and achieve a much higher value.
+  This is the very top of the tent (apex of the cone).
+
+  ### Properties
+
+    (none)
+
+  ### State Variables
+
+    - **color**: (integer) 0=white, 1=color in `rgb` active.
+    - **rgb**: (object) JSON object containing the following keys
+       which describe the apparent color of the light.
+       * `r` - (0--255) apparent red value
+       * `g` - (0--255) apparent green value
+       * `b` - (0--255) apparent blue value
+    - **hsv**: (object) similar to `rgb` but for the HSV colorspace.
+       * `h` - (0--359) apparent hue (in degrees)
+       * `s` - (0--1000) apparent saturation (in permilles)
+       * `v` - (0--1000) apparent value in HSV colorspace (in permilles).
+               Note that `v` and `brightness` are not the same.
+               Users should typically use `brightness` rather than `v`.
+
+  ### Actions
+
+    - **SetRGB({r:int, g:int, b:int})**
+      Set color by RGB values.
+      Light is implicitely turned on if off.
+      A subset of the apparent color components may be specified,
+      in which case unspecified values are left unchanged.
+      Example use case is an integration with seperate R, G, and B sliders,
+      which would do SetRGB(r:int) when the red slider is adjusted.
+
+    - **SetHSV({h:int, s:int, v:int})**
+      Set color by HSV values.
+      Light is implicitely turned on if off.
+      A subset of the `h`, `s`, and `v` parameters may be provided.
+      If `h` or `s` are not provided, they will be unchanged.
+      If `v` is not provided but `s` is provided, then `v` will change
+      so that `brightness` is held fixed.
+
+      The recommended use case is a 2-D color picker where only the `h` and `s`
+      parameters would be supplied, leaving brightness unchanged.
+
+      Due to gamut limitations, the higher `v` values are only
+      rechable with low saturation. Attempting to set a higher-than-possible
+      `v` value will result in trucation to the highest possible value.
+
+      Similarly, when reducing saturation with high `v` value, the
+      `v` will be implicitely reduced so that the desired color (saturation+hue)
+      are achieved at the expense of apparent brightness.
+
+    - **CycleColorPreset()**
+      Change to next color preset.
+      Light is implicitely turned on if off.
+      Different devices will specify different sets of color presets.
+      This is intended to be used with tapping action on "Color" button
+      of RGB SBB products' front-panels.
+
+    - **CycleColor(amount)**
+      Implicitly set maximum saturation and then increase Hue by specified
```

Wrong capitalization?
```suggestion
      Implicitly set maximum saturation and then increase hue by specified
```